Configuration/StandardSequences/python/AlCaRecoStreams_cff.py

```diff
@@ -925,12 +925,12 @@
 	dataTier = cms.untracked.string('ALCARECO')
 	)
 	
-ALCARECOStreamPromptCalibProdPPSDiamondSampicTimingCalib = cms.FilteredStream(
+ALCARECOStreamPromptCalibProdPPSDiamondSampic = cms.FilteredStream(
 	responsible = 'Christopher Misan',
-	name = 'ALCARECOStreamPromptCalibProdPPSDiamondSampicTimingCalib',
+	name = 'ALCARECOStreamPromptCalibProdPPSDiamondSampic',
 	paths  = (pathALCARECOPromptCalibProdPPSDiamondSampicTimingCalib),
```

Comment:
I see that `paths` stays untouched by renaming, is it by purpose ?

Reply:
please see answer in https://github.com/cms-sw/cmssw/pull/38209#issuecomment-1145373710